### PR TITLE
Implement todos in tensorboard docs

### PR DESCRIPTION
**Pull request by rasbt**
There are currently 3 placeholders in the TensorBoard Logging docs. 

I already filled in the first one regarding the progress bar. 

For the images, please let me know where to upload these. They are ~300 kb per file. I can upload them to `lightning/docs/source-pytorch/_static/images/trainer` but not sure if they are too large for that and you prefer hosting them externally. Pls let me know the preference.

For the second one

```
TODO: show chart
```

tensorboard-chart-light

tensorboard-chart-dark

I have two figure candidates 

For the second one 

```
TODO: show single point plotted
```

I have two figure candidates 

tensorboard-singlepoint-light

tensorboard-singlepoint-dark

----
:books: Documentation preview :books:: https://pytorch-lightning--18894.org.readthedocs.build/en/18894/

### CODE REVIEW

**Review by awaelchli — APPROVED** [review]

**Comment by awaelchli** [comment]
We can upload the images to our s3 bucket and then they will be included via link. @Borda maybe you'd like to help here?

**Comment by Borda** [comment]
> @Borda maybe you'd like to help here?

sure, any preference for the file names?

**Comment by rasbt** [comment]
If you don't have a folder structure there, I recommend something like `visualize_logging_basic_tensorboard_chart.png` and `visualize_logging_basic_tensorboard_point.png`

**Comment by rasbt** [comment]
(Just a heads-up but I will be OOO starting tomorrow so pls feel free to merge once the images are inserted)

File: docs/source-pytorch/visualize/logging_basic.rst [hunk 1]
```diff
@@ -86,7 +89,9 @@ Accumulate a metric
 ===================
 When *self.log* is called inside the *training_step*, it generates a timeseries showing how the metric behaves over time.
 
-.. TODO:: show chart
+.. figure:: https://pl-flash-data.s3.amazonaws.com/assets_lightning/logging_basic/visualize_logging_basic_tensorboard_chart.png
```
Comment (Borda) [inline docs/source-pytorch/visualize/logging_basic.rst:92]: Maybe have them in the Pl public bucket instead of Flash, which is already archived...

**Review by Borda — APPROVED** [review]

File: docs/source-pytorch/visualize/logging_basic.rst (hunk 1 above)
Comment (awaelchli) [reply docs/source-pytorch/visualize/logging_basic.rst:92]: What's the name of that bucket? I only know the pl-flash one.

Comment (Borda) [reply docs/source-pytorch/visualize/logging_basic.rst:92]: https://pl-public-data.s3.amazonaws.com/assets_lightning

Comment (awaelchli) [reply docs/source-pytorch/visualize/logging_basic.rst]: Moved, thanks!

**Pull request merged**